File: Package_Manager/1.py
import requests
from bs4 import BeautifulSoup
r = requests.get('https://codingeverybody.github.io/scraping_sample/1.html')
# print(r.text)
soup = BeautifulSoup(r.text, 'html.parser')
print('Title: %s' % soup.title.string)
# Title: sample1
articles = soup.findAll('div', {'class': 'em'})
print('Article: %s' % articles[0].text)
# <div class="em">important information 1</div>
print('======================================')
r = requests.get('https://codingeverybody.github.io/scraping_sample/2.html')
# print(r.text)
soup = BeautifulSoup(r.text, 'html.parser')
print('Title: %s' % soup.title.string)
# Title: sample2
articles = soup.findAll('div', {'class': 'strong'})
print('Article: %s' % articles[0].text)
# <div class="strong">important information 2</div>
print('======================================')

File: Package_Manager/1.rb
require "http"
require "nokogiri"

html = HTTP.get('https://codingeverybody.github.io/scraping_sample/1.html').to_s
page = Nokogiri::HTML(html)
p "Title: "+page.css('title')[0].text
p "Artitle: "+page.css('div.em')[0].text
p "==================================="
html = HTTP.get('https://codingeverybody.github.io/scraping_sample/2.html').to_s
page = Nokogiri::HTML(html)
p "Title: "+page.css('title')[0].text
p "Artitle: "+page.css('div.strong')[0].text
